### PR TITLE
Adding details for API creation using openAPI archive

### PR DESCRIPTION
**Pull request by Sarangan0219**
Adding details about creating API using open API archive.
Related to https://github.com/wso2/carbon-apimgt/pull/8509

### CODE REVIEW

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md [hunk 1]
```diff
@@ -14,8 +14,21 @@ Follow the instructions below to create a REST API using a Open API definition f
 2.  Go to **CREATE API** and Click **I Have an Existing API**.
     ![]({{base_path}}/assets/img/learn/create-a-rest-api-using-existing-swagger-def.jpg)
 
-3.  Select **OpenAPI URL** and provide `http://petstore.swagger.io/v2/swagger.json` as the URL. Click **NEXT** .
-    ![]({{base_path}}/assets/img/learn/create-rest-api-using-swagger-def-form1.jpg)
+3. Select one of the following options:
+
+     * OpenAPI URL - If you select this option, you need to provide an endpoint URL.
+
+     * OpenAPI Archive/File - If you select this option, click Browse and upload either an individual WSDL file or a WSDL archive, which is a WSDL project that has multiple WSDLs.
```
Comment (tgtshanika) [inline en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md:21]: individual Open API definition or an archive with Open API definition with external file referencs - link the external ref spec here
_

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md [hunk 2]
```diff
@@ -14,8 +14,21 @@ Follow the instructions below to create a REST API using a Open API definition f
...
+
+     <html><div class="admonition note">
+     <p class="admonition-title">Note</p>
+     <p>You need to upload an archive if you have remote references in your API definition. When uploading an OpenAPI archive, the master swagger file should be named as **swagger.yaml**/**swagger.json**. If not, the validation will fail at the point of API creation. Referenced files can be named independently. </p>
+     </div>
+     </html>
+
+      As an example, Select **OpenAPI URL** and provide `http://petstore.swagger.io/v2/swagger.json` as the URL.
```
Comment (tgtshanika) [inline en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md:29]: Update the screenshot file

Comment (Sarangan0219) [reply en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md:29]: https://github.com/wso2/docs-apim/pull/1178/commits/b2cf8d94b88e8ece796f98d7cf35d513195c3528

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md (hunk 1 above)
Comment (Sarangan0219) [reply en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md]: https://github.com/wso2/docs-apim/pull/1178/commits/b2cf8d94b88e8ece796f98d7cf35d513195c3528

File: en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md [hunk 3]
```diff
@@ -14,8 +14,21 @@ Follow the instructions below to create a REST API using a Open API definition f
...
+     * OpenAPI Archive/File - If you select this option, click Browse and upload either an individual Open API definition, or an archive with Open API definition with external <a href="https://swagger.io/docs/specification/using-ref/" target="_blank">file references</a>.
+ 
+     <html><div class="admonition note">
+     <p class="admonition-title">Note</p>
+     <p>You need to upload an archive if you have remote references in your API definition. When uploading an OpenAPI archive, the master swagger file should be named as **swagger.yaml**/**swagger.json**. If not, the validation will fail at the point of API creation. Referenced files can be named independently. </p>
+     </div>
+     </html>
+
+      As an example, Select **OpenAPI URL** and provide `http://petstore.swagger.io/v2/swagger.json` as the URL.
+      Click **NEXT** . ![](../../../assets/img/Learn/create-rest-api-using-swagger-def-form1.jpg)
```
Comment (shaniR) [inline en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md:30]: Please use base path instead of relative paths.

**Comment by Mariangela** [comment]
@Sarangan0219, please address the last review comment and also look at resolving the conflicts.

**Comment by Mariangela** [comment]
@shaniR , the merging of this PR is blocked at the moment based on the review comments that you have added. Can you please review it, approve and merge it.

**Pull request merged**